msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -0,0 +1,478 @@
+
+## Test Tool experience in Teams Toolkit
+
+1. Open Visual Studio Code.
+
+1. Select the Teams Toolkit :::image type="icon" source="../assets/images/teams-toolkit-v2/teams-toolkit-sidebar-icon.PNG" border="false"::: icon in the Visual Studio Code **Activity Bar**.
+
+1. Select **Create a New App**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/create-project.png" alt-text="Screenshot shows the location of the Create New Project link in the Teams Toolkit sidebar.":::
+
+1. Select **Bot**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/create-new-app1.png" alt-text="Screenshot shows the Teams Toolkit app templates.":::
+
+1. Select **AI Chat Bot**. If you need a different functionality for your bot, pick a different option.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/ai-chat-bot.png" alt-text="Screenshot shows the app feature to add to your new app.":::
+
+1. Select **JavaScript**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/select-language-tab.png" alt-text="Screenshot shows the option to select the programming language.":::
+
+1. Select **Default folder**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/select-default-location.png" alt-text="Screenshot shows the selection of default location.":::
+
+    You can also change the default location by the following steps:
+
+    1. Select **Browse**.
+
+        :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/select-browse.png" alt-text="Screenshot shows the selection of browse location option.":::
+
+    1. Select the location for project workspace.
+    1. Select the **Select Folder**.
+
+        :::image type="content" source="../assets/images/teams-toolkit-v2/select-folder.png" alt-text="Screenshot shows the folder to select.":::
+
+1. Enter a suitable name for your app and then select the **Enter** key.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/hello-bot.png" alt-text="Screenshot shows where to enter the app name.":::
+
+    A dialog appears, where you need to choose yes or no to trust the authors of the files in this folder.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/first-bot/vsc-trust-authors.png" alt-text="Screenshot shows the dialog to trust or not the authors of the files in this folder.":::
+
+1. From the left pane, select **Run and Debug** (`Ctrl+Shift+D`) and select **Debug in Test Tool** in dropdown list.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/select-debug-in-test-tool.png" alt-text="Screenshot shows the option to select debug in test tool.":::
+
+1. A webpage opens with an Adaptive Card. Type **help** and select **Enter**. The bot displays a list of commands.
```

Comment:
The AI Chat Bot doesn't provide a **help** command, and the snapshot below is from the [test-tool-sample-app](https://github.com/OfficeDev/TeamsFx-Samples/tree/v3/test-tool-sample-app)

Reply:
Updated.